app/src/sharedTest/java/org/oppia/app/help/HelpFragmentTest.kt

```diff
@@ -0,0 +1,91 @@
+package org.oppia.app.help
+
+import androidx.recyclerview.widget.RecyclerView
+import androidx.test.core.app.ActivityScenario.launch
+import androidx.test.espresso.Espresso.onView
+import androidx.test.espresso.action.ViewActions.click
+import androidx.test.espresso.assertion.ViewAssertions.matches
+import androidx.test.espresso.contrib.DrawerActions.close
+import androidx.test.espresso.contrib.DrawerMatchers
+import androidx.test.espresso.contrib.DrawerMatchers.isClosed
+import androidx.test.espresso.contrib.RecyclerViewActions.scrollToPosition
+import androidx.test.espresso.intent.Intents
+import androidx.test.espresso.intent.Intents.intended
+import androidx.test.espresso.intent.matcher.IntentMatchers.hasComponent
+import androidx.test.espresso.matcher.ViewMatchers
+import androidx.test.espresso.matcher.ViewMatchers.withContentDescription
+import androidx.test.espresso.matcher.ViewMatchers.withId
+import androidx.test.espresso.matcher.ViewMatchers.withText
+import androidx.test.ext.junit.runners.AndroidJUnit4
+import org.junit.After
+import org.junit.Before
+import org.junit.Test
+import org.junit.runner.RunWith
+import org.oppia.app.R
+import org.oppia.app.help.faq.FAQActivity
+import org.oppia.app.recyclerview.RecyclerViewMatcher.Companion.atPosition
+import org.oppia.app.recyclerview.RecyclerViewMatcher.Companion.atPositionOnView
+
+@RunWith(AndroidJUnit4::class)
+class HelpFragmentTest {
+  @Before
+  fun setUp() {
+    Intents.init()
+  }
+
+  @Test
+  fun openHelpActivity_scrollRecyclerViewToZeroPosition_showsFAQSuccessfully() {
+    launch(HelpActivity::class.java).use {
+      onView(withId(R.id.help_fragment_recycler_view)).perform(
+        scrollToPosition<RecyclerView.ViewHolder>(
+          0
+        )
+      )
+      onView(
+        atPositionOnView(
+          R.id.help_fragment_recycler_view,
+          0,
+          R.id.recycler_item_text_view
+        )
+      ).check(
+        matches(
+          withText(R.string.frequently_asked_questions_FAQ)
+        )
+      )
+    }
+  }
+
+  @Test
+  fun openHelpActivity_selectFAQ_showFAQActivitySuccessfully() {
+    launch(HelpActivity::class.java).use {
+      onView(atPosition(R.id.help_fragment_recycler_view, 0)).perform(click())
+      intended(hasComponent(FAQActivity::class.java.name))
+    }
+  }
+
+  @Test
+  fun openHelpActivity_openNavigationDrawer_navigationDrawerOpeningIsVerifiedSuccessfully() {
+    launch(HelpActivity::class.java).use {
+      onView(withContentDescription(R.string.drawer_open_content_description)).check(
+        matches(ViewMatchers.isCompletelyDisplayed())
```

Comment:
`ViewMatchers.isCompletelyDisplayed` should be imported directly and so is the `DrawerMachters. isOpen `, as it provides good readability to code.

Something like this:
```
  @Test
  fun openHelpActivity_openNavigationDrawer_navigationDrawerOpeningIsVerifiedSuccessfully() {
    launch(HelpActivity::class.java).use {
      onView(withContentDescription(R.string.drawer_open_content_description)).check(matches(isCompletelyDisplayed()))
        .perform(click())
      onView(withId(R.id.help_fragment_placeholder)).check(matches(isCompletelyDisplayed()))
      onView(withId(R.id.help_activity_drawer_layout)).check(matches(isOpen()))
    }
  }
```

Reply:
Also, once this is done, the imports will also change.

Reply:
Done